### ISSUE TITLE
xhtml not work

### ISSUE THREAD

**Issue by foliovision** [foliovision/freedom-video-player#393]
Original issue link: [#536 xhtml not work](https://github.com/flowplayer/flowplayer/issues/536)

----

<img width='32' height='32' src='https://avatars.githubusercontent.com/u/3122411?v=4' /> [arthurj58](https://github.com/arthurj58) commented on Sep 9, 2013

how work  in  .xhtml

----

<img width='32' height='32' src='https://avatars.githubusercontent.com/u/1395672?v=4' /> [bbbo](https://github.com/bbbo) commented on Sep 9, 2013

Flowplayer is a html5 applciation, it requires <!doctype html> 

----

<img width='32' height='32' src='https://avatars.githubusercontent.com/u/3122411?v=4' /> [arthurj58](https://github.com/arthurj58) commented on Sep 9, 2013

index.xhtml

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" lang="zh-TW" xml:lang="zh-TW">
<head>

   

 <link rel="stylesheet" type="text/css" href="skin/minimalist.css"/>

   
   <style type="text/css">
   body { font: 12px "Myriad Pro", "Lucida Grande", sans-serif; text-align: center; padding-top: 5%; }
   .flowplayer { width: 80%; }
   </style>

   
   <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1/jquery.min.js"></script>

   
   <script type="text/javascript" src="flowplayer.min.js"></script>

</head>
<body>
   
   <div class="flowplayer" data-swf="flowplayer.swf" data-ratio="0.4167">
      <video>
         <source type="video/webm" src="http://stream.flowplayer.org/bauhaus/624x260.webm"/>
         <source type="video/mp4" src="http://stream.flowplayer.org/bauhaus/624x260.mp4"/>
         <source type="video/ogv" src="http://stream.flowplayer.org/bauhaus/624x260.ogv"/>
      </video>
   </div>

</body>
</html>

----

<img width='32' height='32' src='https://avatars.githubusercontent.com/u/1395672?v=4' /> [bbbo](https://github.com/bbbo) commented on Sep 9, 2013

1) please use the formus for this
2) xhtml transitional does not support the <video> tag. The required doctype 
<!doctype html> , at least for som browsers (some ignore the doctype and use to the html5 parser if they see html5 tags, but IE for example does not).

----

<img width='32' height='32' src='https://avatars.githubusercontent.com/u/265438?v=4' /> [nnarhinen](https://github.com/nnarhinen) commented on Sep 9, 2013

Please use the support forums for implementation issues: http://flowplayer.org/forum/

This issue tracker is for issues with the core player itself.
